plugins/modules/content_view_filter_info.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2021 Paul Armstrong
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: content_view_filter_info
+version_added: 3.9.0
+short_description: Fetch information about a Content View Filter
+description:
+  - Fetch information about a Content View Filter
+author:
+  - "Paul Armstrong (@parmstro)"
+options:
+  content_view:
+    description:
+      - name of the content view to add the filter to
+    required: true
+    type: str
+  name:
+    description:
+      - set the name of the content view filter
```

Comment:
this is an info module, so it's not seting anything :)

Reply:
fixed